cli.js

```diff
@@ -23,4 +29,15 @@ const cli = meow(`
 
 cli.flags.app = cli.input.slice(1);
 
-opn(cli.input[0], cli.flags);
+if (cli.input[0]) {
+	opn(cli.input[0], cli.flags);
+} else if (process.stdin.isTTY) {
+	console.error('Input required');
+	process.exit(1);
```

Comment:
Oh you are showing an error. Would rather see it extracted from the this `if-else` block because this looks weird.

1. handle input
2. show error
3. handle input